core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -392,7 +390,7 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
 
   /** Get an optional value, applying variable substitution. */
   private[spark] def getWithSubstitution(key: String): Option[String] = {
-    getOption(key).map(reader.substitute(_))
+    getOption(key).map(reader.substitute)
```

Comment:
In the files I did change, I cleaned up a few other things in nearby code, like this. Other examples are using `.nonEmpty` and removing redundant braces, etc